ctriface/iface.go

```diff
@@ -64,6 +65,8 @@ type StartVMResponse struct {
 
 const (
 	testImageName = "ghcr.io/ease-lab/helloworld:var_workload"
+	fileBackend   = "File"
```

Comment:
please don't use hardcoded values in the modules. have all the hardcoded values in the test's code

Reply:
These 2 const (fileBackend, uffdBackend) should be independent of the test. If UPF enabled, the code pass uffBackend into CreateVM(),  else, the code pass fileBackend into CreateVM().